docs/guides/integration/docker.md

```diff
@@ -16,23 +16,22 @@ description:
 
 ### Running uv in a container
 
-A Docker image is published with a built version of uv available. To run a uv command in a
-container:
+uv provides both *distroful* and *distroless* Docker images. *distroful* images ship operating system files, while *distroless* images ship only the `uv` binaries.
+
+For example, to run a uv command in a container that uses an alpine-based image:
 
 ```console
-$ docker run ghcr.io/astral-sh/uv --help
+$ docker run --rm -it ghcr.io/astral-sh/uv:alpine uv --help
```

Comment:
We shouldn't use alpine because we can't manage Python versions there yet, perhaps Debian?